src/stream/src/from_proto/cdc_filter.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_expr::expr::build_non_strict_from_prost;
+use risingwave_pb::stream_plan::CdcFilterNode;
+
+use super::*;
+use crate::executor::FilterExecutor;
+
+pub struct CdcFilterExecutorBuilder;
+
+/// `CdcFilter` is an extension to the Filter executor
```

Comment:
May update this file to display `CdcFilter` in the dashboard:  
 https://github.com/risingwavelabs/risingwave/blob/2133b82c09d8c23643561fc5cc87f24b2147bad9/dashboard/lib/streamPlan/parser.js#L88